helm/plain-node/values.yaml

```diff
@@ -0,0 +1,30 @@
+testnetName: "coda-testnet"
```

Comment:
[NIT] This could be some generic value without coda or mina in it, maybe just "testnet"